src/main/resources/templates/index.html

```diff
@@ -13,7 +13,7 @@
 
     <ul class="nav">
         <li><a href="index.html">Search</a></li>
-        <li><a href="AddFacility.html">Add Faculty</a></li>
+        <li><a href="addfacility.html">Add Faculty</a></li>
     </ul>
```

Comment:
The name AddFacility.html is inconstant among files leading to a whitepage error